src/components/Sum/Sum.jsx

```diff
@@ -1 +1,7 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <>
+    <p>
+      {`Sum of ${a} and ${b} is ${a + b}`}
+    </p>
+  </>
```

Comment:
You can read more about the use of fragments on the documentation website:
https://react.dev/reference/react/Fragment
```suggestion
    <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
```